chart/dapr-shared/tests/daemonset_test.yaml

```diff
@@ -254,53 +257,53 @@ tests:
       - failedTemplate:
           errorMessage: .Values.shared.daprd.app.protocol is required
 
-  - it: should set --control-plane-address=dapr-api.dapr-system.svc.cluster.local:80 when shared.controlPlane.operatorAddress is empty
+  - it: should set --control-plane-address=dapr-api.dapr-system.svc.cluster.local:80 when shared.controlPlane.operator is empty
     values:
       - ../values.yaml
       - ./values/required.yaml
     set:
-      shared.controlPlane.operatorAddress:
+      shared.controlPlane.operator: {}
     asserts:
       - equal:
-          path: spec.template.spec.containers[0].args[12]
+          path: spec.template.spec.containers[0].args[13]
           value: --control-plane-address=dapr-api.dapr-system.svc.cluster.local:80
 
-  - it: should set --control-plane-address=control.plane.address:80 when shared.controlPlane.operatorAddress is equal to control.plane.address:80
+  - it: should set --control-plane-address=control.plane.address:80 when shared.controlPlane.operator.address is equal to control.plane.address
     values:
       - ../values.yaml
       - ./values/required.yaml
     set:
-      shared.controlPlane.operatorAddress: control.plane.address:80
+      shared.controlPlane.operator.address: control.plane.address
     asserts:
       - equal:
-          path: spec.template.spec.containers[0].args[12]
+          path: spec.template.spec.containers[0].args[13]
           value: --control-plane-address=control.plane.address:80
 
   - it: |
       should set --placement-host-address=dapr-placement-server.dapr-system.svc.cluster.local:50005 
-        when shared.controlPlane.placementServerAddress is empty
+        when shared.controlPlane.placementServer is empty
     values:
       - ../values.yaml
       - ./values/required.yaml
     set:
-      shared.controlPlane.operatorAddress:
+      shared.controlPlane.placementServer: {}
     asserts:
       - equal:
-          path: spec.template.spec.containers[0].args[13]
+          path: spec.template.spec.containers[0].args[14]
           value: --placement-host-address=dapr-placement-server.dapr-system.svc.cluster.local:50005
 
   - it: |
-      should set --placement-host-address=placement.host.address:80 
-        when shared.controlPlane.placementServerAddress is equal to placement.host.address:80
+      should set --placement-host-address=placement.host.address:50005 
+        when shared.controlPlane.placementServer.address is equal to placement.host.address
     values:
       - ../values.yaml
       - ./values/required.yaml
     set:
-      shared.controlPlane.operatorAddress: control.plane.address:80
+      shared.controlPlane.placementServer.address: placement.host.address
     asserts:
       - equal:
-          path: spec.template.spec.containers[0].args[12]
-          value: --control-plane-address=control.plane.address:80
+          path: spec.template.spec.containers[0].args[14]
+          value: --placement-host-address=placement.host.address:50005
```

Comment:
Looks like the test was incorrectly copy-pasted from above and wasn't testing the thing it was supposed to.